ADAL/src/ADAuthenticationError+Internal.h

```diff
@@ -118,6 +126,17 @@
                                        code:(NSInteger)code
                               correlationId:(NSUUID *)correlationId;
 
++ (ADAuthenticationError *)OAuthServerError:(NSString *)protocolCode
+                                description:(NSString *)description
+                                       code:(NSInteger)code
+                              correlationId:(NSUUID *)correlationId
+                                   userInfo:(NSDictionary*)userInfo;
+
+/*! Adds a alternate token to an existing ADAuthentication error's userInfo dictionary */
++ (ADAuthenticationError *)errorFromExistingProtectionPolicyRequiredError:(ADAuthenticationError *)error
+                                                            correlationID:(NSUUID *)correlationId
+                                                                    token:(ADTokenCacheItem*)token;
```

Comment:
As the function is only used by broker, we'd better add a `#if AD_BROKER` to it :)